addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/handler/GardenaThingHandler.java

```diff
@@ -160,7 +159,7 @@ private State convertToState(Device device, ChannelUID channelUID) throws Garden
                     return Boolean.TRUE.toString().equalsIgnoreCase(value) ? OnOffType.ON : OnOffType.OFF;
                 case "DateTime":
                     Calendar cal = DateUtils.parseToCalendar(value);
-                    if (cal != null) {
+                    if (cal != null && cal.get(Calendar.YEAR) > 1970) {
```

Comment:
Hm, isn't this "everything <= 01.01.1971" and not "everything <= 01.01.1970" as intended?

Reply:
Yes, i know, but i thought this is enough. All old dates are not valid, because the gardena system in only a few years old. But if you like, i can do it exactly with <= 01.01.1970

Reply:
I was just pointing out that you didn't implement what you had suggested before :-)
And yes, a reader might wonder what is so special about 1971 - it is imho better to do as you initially suggested.